servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/RealWorld-Application_Angular-Spring-w1/ServiceTemplate.tosca

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <Definitions targetNamespace="http://opentosca.org/example/applications/servicetemplates"
-             id="winery-defs-for_serviceTemplates-RealWorld-Application_Angular-Spring-w1"
+             id="serviceTemplates-RealWorld-Application_Angular-Spring-w1"
```

Comment:
Do we use "serviceTemplates" as prefix in this repository? Or should it be something like "exampleServiceTemplates"? Basically, just a question how to keep it consistent for future repositories containing service templates.

Reply:
Yep, you're right, we should be consistent. Moreover since we might have STs also in other repos which could be used to substitue NTs.